src/lib.rs

```diff
@@ -313,17 +299,12 @@ fn split(hash: u64) -> (u32, u32, u32) {
 ///
 /// assert_eq!(unicode_names2::character("nonsense"), None);
 /// ```
-pub fn character(name: &str) -> Option<char> {
-    // + 1 so that we properly handle the case when `name` has a
-    // prefix of the longest name, but isn't exactly equal.
```

Comment:
why remove this comment?

Reply:
oh, this is mistake. i made no_std incompatible changes and didnt revert it very well